mwtext/filter_functions/wikidata_items_with_wikipedia_sitelinks.py

```diff
@@ -10,13 +11,43 @@ def include(page, revision):
 
     item_doc = json.loads(revision.text)
     qid = item_doc.get('id', None)
+    redirect = item_doc.get('redirect')
     entity = mwbase.Entity.from_json(item_doc)
 
+    # Redirects to other Wikidata item
+    if redirect is not None:
+        return False
+
     # Has a Qid
     if qid is None:
         return False
 
     # Sitelinks to a Wikipedia of any language
-    return any((llink not in ('commonswiki', 'specieswiki') and
+    sitelinks = any((llink not in ('commonswiki',
+                'specieswiki', 'metawiki', 'testwiki') and
                 llink.endswith("wiki"))
                for llink in entity.sitelinks)
+
+    if not sitelinks:
+        return False
+
+    # Is subclass of Wikimedia internal item
+    filepath = resource_filename('mwtext', 'assets/wikimedia_internal_item_qids.txt')
+    with open(filepath) as f:
+        exclude_subclasses = [line.rstrip('\n') for line in f]
```

Comment:
the list of QIDs for this check should probably be passed as an argument to this method. 
I was thinking we should read the file in `process_args` in `transform_content.py` and pass it like:
`transform_content -> is_relevant_page -> include`

What do you say @halfak

Reply:
I think we should just wrap this all up in a custom function inside of this file and rename it to something more specific like "wikidata_articleish_items.py" or something like that.